flask_resty/testing.py

```diff
@@ -97,16 +121,7 @@ def assert_response(response, expected_status_code, expected_data=UNDEFINED):
     else:
         response_data = get_errors(response)
 
-    assert_shape(response_data, expected_data)
-
+    if not isinstance(expected_data, Predicate):
```

Comment:
The idea is that if the user passes in a predicate, then we just use that predicate.

I don't expect this to ever really come up, but it's free.